src/admin/gutenberg/sesamy-post-editor/package.json

```diff
@@ -14,7 +14,11 @@
 		"plugin-zip": "wp-scripts plugin-zip",
 		"start": "wp-scripts start"
 	},
+	"dependencies": {
+		"react": "^18.3.1",
```

Comment:
Added the react dependency. This seems to be what the ai's suggested as fix for the jsx undefined error.